src/Dashboard/Grid.php

```diff
@@ -916,12 +916,10 @@ public function getCardHtml(string $card_id = "", array $card_options = []): str
                 header('Expires: ' . gmdate('D, d M Y H:i:s \G\M\T', time() + $cache_age));
             }
 
-           // server cache
-            if ($GLPI_CACHE->has($cache_key)) {
-                $dashboard_cards = $GLPI_CACHE->get($cache_key);
-                if (isset($dashboard_cards[$gridstack_id])) {
-                    return (string) $dashboard_cards[$gridstack_id];
-                }
+            // server cache
+            $dashboard_cards = $GLPI_CACHE->get($cache_key, []);
+            if (isset($dashboard_cards[$gridstack_id])) {
```

Comment:
Did you miss the null check?
```suggestion
            if ($dashboard_cards !== null && isset($dashboard_cards[$gridstack_id])) {
```

Reply:
As `[]` is used for default value in `$GLPI_CACHE->get()`, `$dashboard_cards` should never be null. Also, unless I am wrong, `isset($not_a_variable['a'])` will return `false` without trigerring any warning.